src/rbac/application/project/controller/project_query_controller.py

```
@@ -0,0 +1,28 @@
+from rbac.domain.project.service.project_query_usecase import ProjectQueryUseCase
+from rbac.utils.slice_content import SliceContent
+
+router = APIRouter(prefix="/project", tags=["project-query"])
+
+
+@cbv(router)
+class ProjectQueryController:
+    @inject
+    def __init__(
+            self,
+            projectQueryUseCase: ProjectQueryUseCase = Depends(Provide[Container.project_query_usecase])
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)